diff --git a/src/test/java/com/example/helloconcourse/WelcomeControllerTest.java b/src/test/java/com/example/helloconcourse/WelcomeControllerTest.java
index a08e889..5aeb24c 100644
--- a/src/test/java/com/example/helloconcourse/WelcomeControllerTest.java
+++ b/src/test/java/com/example/helloconcourse/WelcomeControllerTest.java
@@ -31,7 +31,7 @@ public class WelcomeControllerTest {
                 .getResponse()
                 .getContentAsString();
 
-        assertThat(content).isEqualTo("Broken");
+        assertThat(content).isEqualTo("Hello Tests");
     }
 
 }
